styles.css

```diff
@@ -0,0 +1,216 @@
+.header {
+  grid-area: header;
+  /* background-color: mediumspringgreen; */
+  display: grid;
+  grid-template-columns: 40px 1fr 50px;
+  grid-template-areas: "title   .   toggle-feed";
+  border-bottom: solid grey 2px;
+}
+
+h1 {
+  font-family: "Franklin Gothic Medium", "Arial Narrow", Arial, sans-serif;
+  font-size: 20px;
+  padding-left: 10px;
+}
+
+#title {
+  grid-area: title;
+}
+
+#toggle-icon {
+  grid-area: toggle-feed;
+  padding-top: 10px;
+}
+
+#logo {
+  grid-area: logo;
+  padding-left: 24px;
+  padding-top: 20px;
+}
+
+.main-frame {
+  display: grid;
+  grid-template-columns: 0.5fr 260px 2fr 350px 0.5fr;
+  grid-template-rows: 55px 600px;
+  grid-template-areas:
+    ".   left-side    header      right-side   ."
+    ".   left-side    feed        right-side   .";
+}
+
+.left-side {
+  grid-area: left-side;
+  display: grid;
+  grid-template-rows: 70px 400px;
+  grid-template-areas:
+    "logo"
+    "navigation";
+  border-right: solid grey 2px;
+  /* background-color: lightsteelblue; */
+}
+
+.navigation {
+  display: grid;
+  grid-area: navigation;
+  grid-template-columns: 40px 110px;
+  grid-template-areas: "nav-icons nav-list";
+  padding-left: 26px;
+  padding-top: 25px;
+}
+
+.nav-icons {
+  grid-area: nav-icons;
+  display: grid;
+  grid-template-rows: repeat(7, 30px);
+  gap: 23px;
+}
+
+.nav-list {
+  grid-area: nav-list;
+}
+
+.nav-list ul {
+  list-style-type: none;
+}
+
+li > a {
+  font-family: "Franklin Gothic Medium", "Arial Narrow", Arial, sans-serif;
+  font-size: 20px;
+  text-decoration: none;
+  color: blue;
+}
+
+li {
+  margin-bottom: 30px;
+}
+
+ul {
+  padding: 0;
+  margin-top: 6px;
+}
+
+#Tweeter {
```

Comment:
u could have added the ones with more information at the bottom to show its priority and the ones with less information can be further down